ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/HubTask.groovy

```diff
@@ -38,7 +38,7 @@ abstract class HubTask extends DefaultTask {
 
     @Internal
     JobManager getJobManager() {
-        return new JobManager(getHubConfig().newJobDbClient());
+        return new JobManager(getHubConfig().newJobDbClient(), getHubConfig().newTraceDbClient());
```

Comment:
See above comment. This one is correct. it's using newTraceDbClient() from HubConfig

Reply:
Okay, cool. I'll update the writer for now and switch to using newTraceDbClient